onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -0,0 +1,57 @@
+package org.exoplatform.platform.qa.ui.onlyOffice;
+import com.codeborne.selenide.Condition;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.SiteExplorerHome;
+import org.exoplatform.platform.qa.ui.selenium.platform.ConnectionsManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+public class OnlyOfficeTestIT extends Base {
+
+  HomePagePlatform            homePagePlatform;
+  NavigationToolbar           navigationToolbar;
+  SiteExplorerHome            siteExplorerHome;
+  ManageLogInOut              manageLogInOut;
+  ConnectionsManagement       connectionsManagement;
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+
+    homePagePlatform            =new HomePagePlatform(this);
+    navigationToolbar           =new NavigationToolbar(this);
+    siteExplorerHome            =new SiteExplorerHome(this);
+    manageLogInOut = new ManageLogInOut(this);
+    manageLogInOut.signInCas("root", "gtn");
```

Comment:
We should avoid using root user for tests, it is a special user.

Reply:
ok